app/grandchallenge/api/urls.py

```diff
@@ -69,6 +70,9 @@
     basename="upload-session",
 )
 
+# Component Interfaces
+router.register(r"interfaces", ComponentInterfaceViewSet, basename="interface")
```

Comment:
I would prefer to have the URL as `"components/interfaces"` with `basename="components-interface"`, just in case we want to add an all-components endpoint later (eg, would be useful for pipelines).